### PR TITLE
LPD-45151

### PR DESCRIPTION
**Pull request by JhossephG**
https://liferay.atlassian.net/browse/LPD-45151

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java [hunk 1]
```diff
@@ -106,8 +106,8 @@ public static void setUpClass() throws Exception {
 						StringUtil.startsWith(
 							className,
 							"com.liferay.object.rest.dto.v1_0.ObjectEntry#" +
-								"C_")) {
-
+								"C_")
+						|| className.equals("com.liferay.headless.admin.workflow.dto.v1_0.WorkflowDefinitionLink")) {
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java:110]: @JhossephG use `Objects.equals(`

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:forward:force

**Comment by JhossephG** [comment]
ci:forward:force

**Pull request closed without merging**